R/current_session.R

```diff
@@ -12,6 +13,8 @@
 #'
 #'@export
 current_session = function(){
-	
+	if(session_expired(pkg.env$session)){
+		warning('SB authentication expired, SB interaction may fail. Please re-authentication using authenticate_sb().')
```

Comment:
How about 'Please re-authenticate' instead of 'Please re-authentication'? Otherwise love it.